advice_animal/cli.py

```diff
@@ -273,13 +284,22 @@ def apply(ctx: click.Context, target: str, inplace: bool) -> None:
     for advice_name, result in results.items():
         if result.success:
             if result.modified:
-                click.echo(click.style(advice_name, fg="green") + ": " + result.message)
+                click.echo(
+                    click.style(advice_name, fg="green")
+                    + ": "
+                    + (result.message or "Changes made")
+                )
                 for next_step in result.next_steps:
                     click.echo(click.style(advice_name, fg="yellow") + ": " + next_step)
             else:
                 click.echo(click.style(advice_name, fg="green") + ": No changes needed")
+            return True
```

Comment:
this line seems wrong

Reply:
seems like it will only ever run one advice

Reply:
It return True if the result.success is True. What seems wrong?